cookbook/arlec_grid_connect_smart_plug.rst

```diff
@@ -0,0 +1,132 @@
+Arlec Grid Connect Smart Plug
+=============================
+
+Arlec Grid Connect Smart Plugs are a tuya based smart plugs sold in Australia. 
+
+.. figure:: images/arlec-grid-connect.jpg
+    :align: center
+    :width: 40.0%
+
+These plugs can be flashed with a bit of work.
+
+Flashing
+--------
+
+Prerequisites
+*************
+
+#. Before you begin you'll need:
+
+#. Computer running Windows, Mac or Linux.
+#. Serial to FTDI adapter. (Arduino can also be used.) 
+#. Jumper wires.
+#. Soldering iron. (can be done without but may be harder.)
+
+Connections
+***********
+
+The Arlec Grid connect uses the TYWE2S Tuya module, which can easily be flashed with these connections.
+
+Here is the pinout:
+
+.. figure:: images/tywe2s.jpg
+    :align: center
+    :width: 50.0%
+
+RX -- TX
+
+TX -- RX
+
+3.3v -- 3.3v
+
+GND -- GND
+
+IO0 -- GND (Only needs to be done at boot to enter flashing mode.)
+
+If you are using an Arduino, connect the RST pin to GND to disable the microcontroller and only use as a flashing device.
+
+More information on uploading to the TYWE2S can be found here: https://github.com/arendst/Sonoff-Tasmota/wiki/CE-Smart-Home---LA-WF3-Wifi-Plug-(TYWE2S)
```

Comment:
This is a dead link? https://github.com/arendst/Sonoff-Tasmota/wiki/CE-Smart-Home---LA-WF3-Wifi-Plug-